src/components/search/advancedSearch/advancedSearchContext.js

```diff
@@ -26,144 +38,50 @@ export function useAdvancedSearchContext() {
   return useContext(AdvancedSearchContext);
 }
 
-function dropdownReducer(prev, current) {
-  return prev?.map((singleDropdownIndex) => {
-    if (current.indexName === singleDropdownIndex.searchIndex) {
-      return {
-        searchIndex: current.indexName,
-        value: current.menuItemsState
-          .filter((item) => item.isSelected === true)
-          .map((item) => item.value),
-      };
-    } else {
-      return singleDropdownIndex;
-    }
-  });
-}
-
-const initialInputFields = [
-  { value: "", prefixLogicalOperator: null, searchIndex: "term.default" },
-  {
-    value: "",
-    prefixLogicalOperator: LogicalOperatorsEnum.AND,
-    searchIndex: "term.title",
-  },
-];
-
 export default function AdvancedSearchProvider({ children }) {
   // TODO: Move this part of the state into Context
   //  So we are able to control default view from context as well
-  // const router = useRouter();
-  // const { page: pageNo = 1 } = router.query;
-  // const cql = router?.query?.cql || null;
-  // let fieldSearch = router?.query?.fieldSearch;
-  // if (fieldSearch) {
-  //   fieldSearch = JSON.parse(fieldSearch);
-  // }
-
-  //prefixLogicalOperator is an enum of AND, OR , NOT
-  /** @typedef {("AND"|"OR"|"NOT"|null)} PrefixLogicalOperator */
-  /** @typedef {{value: string, prefixLogicalOperator: PrefixLogicalOperator, searchIndex: string}} InputField */
-  const [/** @type Array.<InputField> */ inputFields, setInputFields] =
-    useState(initialInputFields);
-
-  const [dropdownInitState, setDropdownInitState] = useState([]);
+  const router = useRouter();
+  const { page = "1", cql: cqlFromUrl = null, fieldSearch = "" } = router.query;
+  const fieldSearchFromUrl = fieldSearch && JSON.parse(fieldSearch);
```

Comment:
Ligner at fieldSearch ikke kan være null. Man kan måske sætte default til null istedet for "" . 
For der er allerede et tjek for `fieldSearch &&`